vue/mustache-interpolation-spacing still running if v-pre is set on node

<!--
  Before posting the issue, please confirm that the problem you're getting
  is not related with your code editor configuration.
  To make sure it's not, run: yarn eslint src/your-file.vue
-->

**Tell us about your environment**	
* **ESLint version: v5.6.0**	
* **eslint-plugin-vue version: v4.7.1**	
* **Node version: v10.13.0**	

**Please show your full configuration:**
<!-- Paste content of your .eslintrc file -->
```js
// https://eslint.org/docs/user-guide/configuring

module.exports = {
    root: true,
    parserOptions: {
        parser: 'babel-eslint'
    },
    env: {
        browser: true,
    },
    extends: [
        // https://github.com/vuejs/eslint-plugin-vue#priority-a-essential-error-prevention
        'plugin:vue/recommended',
    ],
    // required to lint *.vue files
    plugins: [
        'vue'
    ],
    // add your custom rules here
    rules: {
        "indent": ["error", 4],
        "quotes": ["error", "double"],
        "semi": ["error", "always"],
        "comma-dangle": ["error", "only-multiline"],
        // allow async-await
        'generator-star-spacing': 'off',
        // allow debugger during development
        'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
        "vue/html-indent": ["error", 4, {
            "attribute": 1,
            "closeBracket": 0,
            "alignAttributesVertically": true,
            "ignores": []
        }],
        "vue/html-self-closing": ["error", {
            "html": {
                "void": "never",
                "normal": "any",
                "component": "always"
            },
            "svg": "always",
            "math": "always"
        }]
    }
};
```

**What did you do?**	
I ran ESLint with no parameters on the following code within a .vue file:
```vue
<b>Response message</b> - You can use <code v-pre>{{sender}}</code> or <code v-pre>{{mentioned}}</code> variables.
```

**What did you expect to happen?**
ESLint passes

**What actually happened?**
ESLint gives error. It failed on `vue/mustache-interpolation-spacing` despite v-pre being on the element.
```
  28:71   error  Expected 1 space after '{{', but not found   vue/mustache-interpolation-spacing
  28:79   error  Expected 1 space before '}}', but not found  vue/mustache-interpolation-spacing
  28:104  error  Expected 1 space after '{{', but not found   vue/mustache-interpolation-spacing
  28:115  error  Expected 1 space before '}}', but not found  vue/mustache-interpolation-spacing
```

--- a/package.json
+++ b/package.json
@@ -46,7 +46,7 @@
     "eslint": "^5.0.0"
   },
   "dependencies": {
-    "vue-eslint-parser": "^4.0.2"
+    "vue-eslint-parser": "^5.0.0"
   },
   "devDependencies": {
     "@types/node": "^4.2.16",

--- a/tests/lib/rules/no-unused-vars.js
+++ b/tests/lib/rules/no-unused-vars.js
@@ -36,10 +36,10 @@ tester.run('no-unused-vars', rule, {
       code: '<template><ol v-for="i in data"><li v-for="f in i">{{ f.bar.baz }}</li></ol></template>'
     },
     {
-      code: '<template scope="props">{{props}}</template>'
+      code: '<template><template scope="props">{{props}}</template></template>'
     },
     {
-      code: '<template scope="props"><span v-if="props"></span></template>'
+      code: '<template><template scope="props"><span v-if="props"></span></template></template>'
     },
     {
       code: '<template><div v-for="(item, key) in items" :key="key">{{item.name}}</div></template>'
@@ -61,7 +61,15 @@ tester.run('no-unused-vars', rule, {
       errors: ["'props' is defined but never used."]
     },
     {
-      code: '<template><template v-for="i in 5"><comp v-for="j in 10">{{i}}{{i}}</comp></template></template>',
+      code: '<template><span slot-scope="props"></span></template>',
+      errors: ["'props' is defined but never used."]
+    },
+    {
+      code: '<template><span><template scope="props"></template></span></template>',
+      errors: ["'props' is defined but never used."]
+    },
+    {
+      code: '<template><div v-for="i in 5"><comp v-for="j in 10">{{i}}{{i}}</comp></div></template>',
       errors: ["'j' is defined but never used."]
     },
     {

--- a/tests/lib/rules/no-unused-vars.js
+++ b/tests/lib/rules/no-unused-vars.js
@@ -57,11 +57,11 @@ tester.run('no-unused-vars', rule, {
       errors: ["'i' is defined but never used."]
     },
     {
-      code: '<template scope="props"></template>',
+      code: '<template><template scope="props"></template></template>',
       errors: ["'props' is defined but never used."]
     },
     {
-      code: '<template v-for="i in 5"><comp v-for="j in 10">{{i}}{{i}}</comp></template>',
+      code: '<template><template v-for="i in 5"><comp v-for="j in 10">{{i}}{{i}}</comp></template></template>',
       errors: ["'j' is defined but never used."]
     },
     {

--- a/tests/lib/rules/valid-v-else-if.js
+++ b/tests/lib/rules/valid-v-else-if.js
@@ -43,7 +43,7 @@ tester.run('valid-v-else-if', rule, {
   invalid: [
     {
       filename: 'test.vue',
-      code: '<template v-else-if="foo"><div></div></template>',
+      code: '<template><template v-else-if="foo"><div></div></template></template>',
       errors: ["'v-else-if' directives require being preceded by the element which has a 'v-if' or 'v-else-if' directive."]
     },
     {

--- a/tests/lib/rules/valid-v-else.js
+++ b/tests/lib/rules/valid-v-else.js
@@ -43,7 +43,7 @@ tester.run('valid-v-else', rule, {
   invalid: [
     {
       filename: 'test.vue',
-      code: '<template v-else><div></div></template>',
+      code: '<template><template v-else><div></div></template></template>',
       errors: ["'v-else' directives require being preceded by the element which has a 'v-if' or 'v-else' directive."]
     },
     {

--- a/tests/lib/rules/valid-v-for.js
+++ b/tests/lib/rules/valid-v-for.js
@@ -90,24 +90,30 @@ tester.run('valid-v-for', rule, {
     {
       filename: 'test.vue',
       code: `
-        <template v-for="x in xs">
-          <template v-for="y in x.ys">
-            <li v-for="z in y.zs" :key="z.id">
-              123
-            </li>
+        <template>
+          <template v-for="x in xs">
+            <template v-for="y in x.ys">
+              <li v-for="z in y.zs" :key="z.id">
+                123
+              </li>
+            </template>
           </template>
-        </template>`
+        </template>
+      `
     },
     {
       filename: 'test.vue',
       code: `
-        <template v-for="x in xs">
-          <template v-for="y in ys">
-            <li v-for="z in zs" :key="x.id + y.id + z.id">
-              123
-            </li>
+        <template>
+          <template v-for="x in xs">
+            <template v-for="y in ys">
+              <li v-for="z in zs" :key="x.id + y.id + z.id">
+                123
+              </li>
+            </template>
           </template>
-        </template>`
+        </template>
+      `
     }
   ],
   invalid: [
@@ -215,37 +221,46 @@ tester.run('valid-v-for', rule, {
       filename: 'test.vue',
       errors: ["Expected 'v-bind:key' directive to use the variables which are defined by the 'v-for' directive."],
       code: `
-        <template v-for="x in xs">
-          <template v-for="y in a.ys">
-            <li v-for="z in y.zs" :key="z.id">
-              123
-            </li>
+        <template>
+          <template v-for="x in xs">
+            <template v-for="y in a.ys">
+              <li v-for="z in y.zs" :key="z.id">
+                123
+              </li>
+            </template>
           </template>
-        </template>`
+        </template>
+      `
     },
     {
       filename: 'test.vue',
       errors: ["Expected 'v-bind:key' directive to use the variables which are defined by the 'v-for' directive."],
       code: `
-        <template v-for="x in xs">
-          <template v-for="y in x.ys">
-            <li v-for="z in a.zs" :key="z.id">
-              123
-            </li>
+        <template>
+          <template v-for="x in xs">
+            <template v-for="y in x.ys">
+              <li v-for="z in a.zs" :key="z.id">
+                123
+              </li>
+            </template>
           </template>
-        </template>`
+        </template>
+      `
     },
     {
       filename: 'test.vue',
       errors: ["Expected 'v-bind:key' directive to use the variables which are defined by the 'v-for' directive."],
       code: `
-        <template v-for="x in xs">
-          <template v-for="y in x.ys">
-            <li v-for="z in x.zs" :key="z.id">
-              123
-            </li>
+        <template>
+          <template v-for="x in xs">
+            <template v-for="y in x.ys">
+              <li v-for="z in x.zs" :key="z.id">
+                123
+              </li>
+            </template>
           </template>
-        </template>`
+        </template>
+      `
     }
 
   ]

--- a/tests/lib/rules/valid-v-for.js
+++ b/tests/lib/rules/valid-v-for.js
@@ -77,11 +77,11 @@ tester.run('valid-v-for', rule, {
     },
     {
       filename: 'test.vue',
-      code: '<template v-for="x of list"><slot name="item" /></template>'
+      code: '<template><template v-for="x of list"><slot name="item" /></template></template>'
     },
     {
       filename: 'test.vue',
-      code: '<template v-for="x of list">foo<div></div></template>'
+      code: '<template><template v-for="x of list">foo<div></div></template></template>'
     },
     {
       filename: 'test.vue',
@@ -262,6 +262,5 @@ tester.run('valid-v-for', rule, {
         </template>
       `
     }
-
   ]
 })

